packages/plugin-sass/src/plugin-sass.ts

```diff
@@ -121,10 +123,12 @@ export function sass({
                     cacheDependencies: ['node-sass'],
                   })),
                   {
-                    path: 'sass-loader',
+                    loader: 'sass-loader',
                     options: {
-                      sourceMap: sourceMaps,
-                      includePaths: sassIncludePaths,
+                      sassOptions: {
+                        sourceMap: sourceMaps,
```

Comment:
I think `sourceMaps` is still a top-level option (https://webpack.js.org/loaders/sass-loader/#options)